app-header/app-header.html

```diff
@@ -95,6 +95,9 @@
       position: relative;
       display: block;
       -webkit-transform: translateZ(0);
+      transition-property: -webkit-transform;
+      transition-property: transform;
```

Comment:
Btw, only Safari 8 needs -webkit prefix for transform.  So I am guessing you will see the problem when running on Safari 8.